### DISCUSSION TITLE
How to search in jsonb column for any text?

### DISCUSSION THREAD

**Discussion by nappalm** [supabase/supabase#12677] [General]
I've got lost in the documentation, I have a json column in my notes table, I'm applying the following filter:

```
 const {error, data} = await supabase
      .from(tables.NOTES)
        .select('*')
        .filter('content::text', 'ilike', '%Feb 2023%');
```

the error I have is the following:
```
{
  "code": "42883",
  "details": null,
  "hint": "No operator matches the given name and argument types. You might need to add explicit type casts.",
  "message": "operator does not exist: jsonb ~~ unknown"
}
```

It should be said that I use the following query in supabase and it works correctly:
`SELECT * FROM notes WHERE content::text ILIKE '%Feb 2023%'
`

Does anyone know how I improve my query to solve a search problem?

**Comment by steve-chavez** [comment]
Try with computed columns as mentioned here https://github.com/supabase/supabase/discussions/3582#discussioncomment-1497587
